AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -10065,6 +10065,8 @@ privacy.lecker.de#%#AG_onLoad(function(){if(!document.cookie.includes("euconsent
 privacy.madame.de#%#AG_onLoad(function(){if(!document.cookie.includes("euconsent-v2")){var g=new MutationObserver(function(){setTimeout(function(){var b=document.querySelector('button.no-children[path="\[0,2,1\]"]');b&&b.click();},300);setTimeout(function(){var c=document.querySelector('div[title="Berechtigte Interessen"]'); c&&c.click();},300);setTimeout(function(){var d=document.querySelectorAll('label[aria-checked="true"] > .off'); d.forEach(function(d){d.click();});},800); var f=document.querySelector('button.no-children[path="\[0,3,0\]"]'); setTimeout(function(){f&&f.click();},1000);});g.observe(document,{childList:!0,subtree:!0});setTimeout(function(){g.disconnect()},3E4)}});
 consent.unilad.com,consent.ladbible.com,consent.gamingbible.co.uk,consent.unilad.co.uk#%#AG_onLoad(function(){if(window.location.href.includes("consentUUID")){var g=new MutationObserver(function(){var b=document.querySelector('button[title*="Accept" i]');b&&(g.disconnect(),b.click())});g.observe(document,{childList:!0,subtree:!0});setTimeout(function(){g.disconnect()},1E4)}});
 !
+spoint.spox.com#%#AG_onLoad(function(){if(-1==document.cookie.indexOf('spx')){var a=new MutationObserver(function(){var b=document.querySelector('.message-component button[title="Accept"]');b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
+spoint.spox.com#%#AG_onLoad(function(){if(-1==document.cookie.indexOf('spx')){var a=new MutationObserver(function(){var b=document.querySelector('.message-component button[title="Zustimmen"]');b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
```

Comment:
I think that the JS rules can be merged into one rule because such type of elements must be existed only one element.
```suggestion
spoint.spox.com#%#AG_onLoad(function(){if(-1==document.cookie.indexOf('spx')){var a=new MutationObserver(function(){var b=document.querySelector('.message-component button[title="Accept"], .message-component button[title="Zustimmen"]');b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
```
Suggested change: `.message-component button[title="Accept"]` and `.message-component button[title="Zustimmen"]` => `.message-component button[title="Accept"], .message-component button[title="Zustimmen"]`